panel/tests/ui/layout/test_feed.py

```diff
@@ -129,9 +135,12 @@ def test_feed_scroll_to_latest_within_limit(page):
 
     feed.scroll_to_latest(scroll_limit=100)
 
-    wait_until(lambda: feed_el.evaluate(
-        '(el) => el.scrollHeight - el.scrollTop - el.clientHeight'
-    ) == 0, page)
+    def case():
```

Comment:
This seems exactly equivalent.

Reply:
It is functionality-wise, but pytest does its magic, and you get the value of the assert output, instead of a TimeoutError. 

![image](https://github.com/user-attachments/assets/aa050663-306a-4861-8623-9b48875cfed9)

Adding the new feed was to try to slow things down, I don't think it is ideal.

Reply:
Ah okay, I've tried and failed to make pytest understand wait_until properly. Would be nice to find a general solution.

Reply:
I think this is the best solution, this will rewrite the function inside the test.